ecl2df/common.py

```diff
@@ -84,7 +85,7 @@ def write_dframe_stdout_file(
         caller_logger (logging): Used if not stdout
         logstr (str): Logged if not stdout.
     """
-    if output == "-":
+    if output == __MAGIC_STDOUT__:
```

Comment:
I think `None` would work just as well here? `-` usually is `stdin` in bash, might cause some confusion?

Reply:
Also, isn't `-` a perfectly valid file name?

Reply:
It works, but it creates a lot of problems. Many bash tools cannot create the file, `touch -` will not work. `echo foo > -` works, but `cat -` will not work afterwards. Not even `cat -- -`. You can also try `mkdir -` but it is hard to `cd -` afterwards.